src/components/charts/google/GoogleChart.jsx

```diff
@@ -72,7 +58,7 @@ let GoogleChart = React.createClass({
   _initializeChart() {
     this.adapter = new ChartDataGoogleDataAdapter(this.props.chartData);
 
-    this.chart = new this.props.googleChart(this.getDOMNode());
+    this.chart = new this.props.GoogleChart(this.getDOMNode());
```

Comment:
you cannot modify case of proptypes like .... it will break components that uses it ...